tools/apksize/proguard-rules-aggressive.pro

```diff
@@ -0,0 +1,20 @@
+# Add project specific ProGuard rules here.
```

Comment:
It the entire file is commented, we can skip wiring this in

Reply:
I've removed the comments. The file does actually disable warnings. So I've left it included.